server/hydrogen-render/2-render-hydrogen-to-string-fork-script.js

```diff
@@ -13,12 +15,31 @@ process.on('message', async (options) => {
   try {
     const resultantHtml = await _renderHydrogenToStringUnsafe(options);
 
-    // Send back the data we need
-    process.send({
-      data: resultantHtml,
+    assert(resultantHtml, `No HTML returned from _renderHydrogenToStringUnsafe.`);
+
+    // Send back the data we need to the parent.
+    await new Promise((resolve, reject) => {
```

Comment:
This is wrapped in a promise because `process.send(...)` is async and only has a `callback` option. And we want any error from the callback to be caught by our general `try`/`catch` out here